### ISSUE TITLE
Data Cache is not available for Read Replica of MySql in Enterprise Plus edition

### ISSUE THREAD

**Issue by AmarendraSingh88** [terraform-google-modules/terraform-google-sql-db#631]
### TL;DR

Data Cache is not available for Read Replica of MySql in Enterprise Plus edition and because of that we are not able to use Enterprise Plus upgrade via terraform.

### Expected behavior

We should have the data_cache_enabled parameter available for MySql Read Replica as well, the same way we have it for Master MySql database.

### Observed behavior

We do not have data_cache_enabled param available for [MySql Read Replica](https://github.com/terraform-google-modules/terraform-google-sql-db/blob/master/modules/mysql/read_replica.tf) while it is there for [Master Database](https://github.com/terraform-google-modules/terraform-google-sql-db/blob/master/modules/mysql/main.tf#L102)

### Terraform Configuration

```hcl
data_cache_enabled    = var.data_cache_enabled
```

### Terraform Version

```sh
1.4.5
```

### Additional information

[This](https://github.com/terraform-google-modules/terraform-google-sql-db/pull/491) PR is there, which added the edition and data cache related details but seems it is missed for MySql Read Replica.

**Comment by AmarendraSingh88** [comment]
Hello team,
Any plan to include this in the next release?

**Comment by imrannayer** [comment]
asked developed of PR #642 to add it in his PR.

**Comment by AmarendraSingh88** [comment]
Hello @imrannayer , 
I see that the above PR is already merged and didn't include the changes for MySql. Would it be possible for you to add the changes?
Thanks

**Comment by imrannayer** [comment]
@AmarendraSingh88 PR #652 will fix it
